api/src/main/java/jakarta/resource/spi/ManagedConnection.java

```diff
@@ -195,7 +196,6 @@ void removeConnectionEventListener(
    *  @return     XAResource instance
    *
    *  @throws     ResourceException     generic exception if operation fails
-   *  @throws     NotSupportedException if the operation is not supported
```

Comment:
Was this removed by mistake?

Reply:
No it's not in the method signature so javadoc complains

Reply:
It isn't valid to remove this.  See my reply https://github.com/eclipse-ee4j/jca-api/pull/106#discussion_r471574936 for details.

Reply:
> No it's not in the method signature so javadoc complains

What does it complain about? I suppose "reference not found" as `NotSupportedException` is not imported. So perhaps using FQN would silent javadoc?

Reply:
It complains that there is no throws for NotSupportedException in the method signature

Reply:
> It complains that there is no throws for NotSupportedException in the method signature

On current master (19e0dcd72d4ca31c6e9c03d85f9ba101a5cbb381), with maven 3.6.3 and 1.8.0_261 - with javadoc plugin bumped as in this PR 3.2.0 I'm getting
```
[ERROR] /root/jca-api/api/src/main/java/jakarta/resource/spi/ManagedConnection.java:198: error: reference not found
[ERROR]    *  @throws     NotSupportedException if the operation is not supported
[ERROR]       ^
[ERROR] /root/jca-api/api/src/main/java/jakarta/resource/spi/ManagedConnection.java:212: error: reference not found
[ERROR]    *  @throws     NotSupportedException if the operation is not supported
[ERROR]       ^
[ERROR] /root/jca-api/api/src/main/java/jakarta/resource/spi/ManagedConnection.java:227: error: reference not found
[ERROR]    *  @throws     NotSupportedException if the operation is not supported
```
It that what you get as well?

Reply:
Yes

Reply:
> Yes

As I said - it's due to `NotSupportedException` being unknown at this place in javadoc comment.
Import type (and it will become import not used in code) or use FQN in tag and error will be gone.
And documentation will still cover more specific exception (as `NSE` extends `ResourceException`).

Reply:
OK I will try that